src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,31 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private final Random random = new Random();
+    private final ColorSupplier colorSupplier = new ColorSupplier();
+
+    public Figure getDefaultFigure() {
+        return new Circle("WHITE", 10);
+    }
+
+    public Figure getRandomFigure() {
+        int randomNumberFirst = random.nextInt(1, 10);
+        int randomNumberSecond = random.nextInt(1, 10);
+        int randomNumberThird = random.nextInt(1,10);
```

Comment:
Consider using constants for the minimum and maximum bounds of the random numbers to enhance code readability and maintainability. For example, `private static final int MIN_BOUND = 1; private static final int MAX_BOUND = 10;`

Reply:
magic numbers